packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -501,4 +487,113 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
       throw new WalletSignMessageAndVerifyError(errMsg).message;
     }
   }
+
+  async prepRawTransaction(input: RawTransactionPrepPayload): Promise<AnyRawTransaction> {
```

Comment:
can we add some comments on what this function does

Reply:
Oof, I forgot documentation in my mad rush, let me fix that